wdl-format/src/config.rs

```diff
@@ -0,0 +1,21 @@
+//! Configuration for formatting.
```

Comment:
```suggestion
//! Formatting configuration.
```

Reply:
nit: the module documentation and the `Config` documentation shouldn't be identical.